pkg/pop/poptoken.go

```diff
@@ -0,0 +1,178 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Seeing some file with this header, other without. Should we remove this @weinong ?

Reply:
sure. let's remove that for consistency

Reply:
Removed the header from code files